__tests__/app.test.js

```diff
@@ -73,4 +72,55 @@ describe('GET /api/reviews/:review_id', () => {
             expect(res.body.msg).toBe('invalid endpoint syntax')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/reviews/:review_id/comments', () => {
+    test('GET - STATUS: 200 - responds with an array of the chosen review_id with specified properties', () => {
+        return request(app).get('/api/reviews/3/comments')
+        .expect(200)
+        .then((res) => {
+            expect(Array.isArray(res.body.comments)).toBe(true);
+            res.body.comments.forEach((comment) => {
```

Comment:
Whenever using a forEach() that contains assertions in a test, make sure you check for length before invoking the iteration. Otherwise, if the array is empty you will get a false positive test result.